parsers/nsis.c

```diff
@@ -216,7 +216,7 @@ static const unsigned char* parseLangString (const unsigned char* cp, vString *n
 {
 	cp = skipWhitespace (cp);
 
-	/* `^' is not explaned the nsis reference manual. However, it is used
+	/* `^' is not explained the nsis reference manual. However, it is used
```

Comment:
```suggestion
	/* `^' is not explained in the nsis reference manual. However, it is used
```
or maybe "mentioned" instead of "explained"?

Reply:
Updated.